index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9650a62e47.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <!-- PAGE ABOUT -->
+  <header id="about">
+    <nav id="navbar">
+      <ul>
+        <li><a href="#about">About</a></li>
+        <li><a href="#work">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+    <div id="welcome-section" class="container">
+      <h1 id="title">Hey! This is Rifat & Fred</h1>
+      <p class="lead"><em>We are full stack developer students of Microverse professional
+        bootcamp!</em></p>
+    </div>
+  </header>
+
+  <!-- PROJECT SECTION -->
+  <section id="projects">
+    <div class="project-tile">
+      <h1 id="caption">"These are some of my projects"</h1><br>
+      <div id="bottom-line"></div>
+      <div class="projects">
+        <div class="box">
+          <a target="_blank" href="">
+            <img src="https://images.pexels.com/photos/546819/pexels-photo-546819.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="project-img" class="project-img">
+            <p>
+              <span class="code">&lt;</span>
+              Tribute Page
+              <span class="code">/&gt;</span>
+            </p>
+          </a>
+        </div>
+
+        <div class="box">
+          <a target="_blank" href="" alt="project-img" class="project-img">
+            <img src="https://images.pexels.com/photos/546819/pexels-photo-546819.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="project-img" class="project-img">
+            <p>
+              <span class="code">&lt;</span>
+              Personal Portfolio
+              <span class="code">/&gt;</span>
+            </p>
+          </a>
+        </div>
+
+        <div class="box">
+          <a target="_blank" href="">
+            <img src="https://images.pexels.com/photos/546819/pexels-photo-546819.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="project-img" class="project-img">
+            <p>
+              <span class="code">&lt;</span>
+              Documentation Page
+              <span class="code">/&gt;</span>
+            </p>
+          </a>
+        </div>
+
+
+
+        
+  </section>
+
+  <!-- CONTACT SECTION -->
+
+
+  <!-- FOOTER SECTION -->
+  <footer id="main-footer">
+    <div class="container">
+      <p>&copy; Copyright Rifat and Fred
+      </p>
+      <p id="creator">This site recreated by <a href="https://github.com/Rifat8080">Rifat & fred</a></p>
+    </div>
+  </footer>
+</body>
+<script src=""></script>
```

Comment:
Good implementation of code in your webpage but I have noticed there is no linking source file for js. You should link js files to your HTML code to make your website more responsive😎😂